### ISSUE TITLE
Why these childProps?

### ISSUE THREAD

**Issue by davidascher** [founderlab/react-dropzone-s3-uploader#9]
When I use this (awesome btw) component with something like:

```
      <div style={box}>
        <DropzoneS3Uploader onFinish={this.handleFinishedUpload} {...uploaderProps}>
          <div style={legend}>
            Drop an image on here or click to select
          </div>
        </DropzoneS3Uploader>
      </div>
```

the rendered component looks fine, but React complains:

```
Warning: Unknown props `fileUrl`, `s3Url`, `filename`, `progress`, `error`, `imageStyle` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
```

Which makes some sense.

Digging in, I see [this](https://github.com/founderlab/react-dropzone-s3-uploader/blob/master/src/DropzoneS3Uploader.js#L137) line, which maybe makes sense for the `FileComponent` path, but I don't think it makes sense for the `this.props.children` path.

I'm happy to submit a PR if I'm not missing a scenario.

**Comment by Akronix** [comment]
Hi!
I agree with @davidascher. I also have a `<div>` as children and React warns about "Unknown props".
If those props are somehow required in some cases to be passed down to children, maybe they can be activated through a propagateProps boolean prop or something like that.

**Comment by gwilymhumphreys** [comment]
Hey, cheers. I agree too. Passing them down to props.children is left over from a time before the fileComponent prop (when React wasn't warning about superfluous props). We don't need to do so any more, I'll remove it.

**Comment by Akronix** [comment]
hi! any news here?

**Comment by gwilymhumphreys** [comment]
Published v0.8.0, which doesn't pass them through to its children.

**Comment by Akronix** [comment]
Great! thank you @gwilymhumphreys !! :tada:
